### DISCUSSION TITLE
Array of configurable trigger events

### DISCUSSION THREAD

**Discussion by paulineribeyre** [community/community#25893] [Actions]
<p>The documentation about the “on” keyword states:<br>
“You can provide a single event string, array of events, array of event types, or an event configuration map that schedules a workflow or restricts the execution of a workflow”.</p>
<p>Is there a way to use an array of events and configuration maps at the same time? I suspect an “array of event types” is what I need, but I don’t know what the syntax is.</p>
<p>For example, I haven’t found a way to do this:</p>
<pre><code>on:
- pull_request
- issue_comment:
    types: [created, edited]
</code></pre>
<p>Only “on: [pull_request, issue_comment]” is accepted.</p>
<p>Another example of something I would like to do:</p>
<pre><code>on:
- push
- schedule:
  - cron: 0 10 * * *
</code></pre>
<p>To do this, I had to create 2 identical workflows with different trigger events.</p>

**Comment by wipe2238** [comment, marked as answer]
<p>Try</p>
<pre><code>on:
 push:
 schedule:
 - cron: */15 * * * *
</code></pre>
